h/app.py

```diff
@@ -97,7 +97,7 @@ def create_api(global_config, **settings):
     config.set_authorization_policy(acl_authz)
     config.set_root_factory('h.api.resources.create_root')
 
-    config.add_renderer('json', JSON(indent=4))
+    config.add_renderer('json', JSON())
```

Comment:
You can just remove this line then. The 'json' renderer is a default and I was just overriding it here.